pkg/util/git/git.go

```diff
@@ -64,7 +65,14 @@ func (gr *GitCLIRepository) Clone(ctx context.Context, options CloneOptions) err
 		}
 
 		args = append(args, options.Args...)
-		out, err := command.CombinedOutput(ctx, gr.LocalPath, expand.ListEnviron(os.Environ()...), "git", args...)
+		os.Setenv("GIT_SSH_COMMAND", "ssh -oBatchMode=yes")
```

Comment:
Is this to prevent prompting for a password? If not, a comment with explanation might be helpful.

Reply:
Yes, this is for preventing prompts and preventing ssh from asking for adding unknown hosts to known host list. I'll add comments there.

Reply:
Done.